File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--manage-control-layout-on-actions-panes_4.vb
    Private Sub ActionsPane_OrientationChanged(ByVal sender As Object, ByVal e As System.EventArgs) _
        Handles ActionsPane.OrientationChanged

        If Me.ActionsPane.Orientation = Orientation.Horizontal Then

            Me.ActionsPane.StackOrder = Microsoft.Office.Tools.StackStyle.FromLeft
        Else
            Me.ActionsPane.StackOrder = Microsoft.Office.Tools.StackStyle.FromTop
        End If
    End Sub

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--manage-control-layout-on-actions-panes_4.cs
        private void ThisDocument_Startup(object sender, System.EventArgs e)
        {
            this.ActionsPane.OrientationChanged += new EventHandler(ActionsPane_OrientationChanged);
        }

        private void ActionsPane_OrientationChanged(object sender, EventArgs e)
        {
            if (ActionsPane.Orientation == Orientation.Horizontal)
            {
                this.ActionsPane.StackOrder = Microsoft.Office.Tools.StackStyle.FromLeft;
            }
            else
            {
                this.ActionsPane.StackOrder = Microsoft.Office.Tools.StackStyle.FromTop;
            }
        }